### PR TITLE
feat(api): Add `refreshed` attribute to ApiToken

### PR DESCRIPTION
**Pull request by mnoble**
Adds a boolean attribute to `ApiToken` to denote whether it has been refreshed or not.

Once a token is refreshed, it should no longer be allowed to be refreshed again. It should be completely inactive.

### CODE REVIEW

**Comment by wedamija** [comment]
Does it make sense at all to just delete the token after it has been refreshed, or is otherwise inactive?

**Comment by mnoble** [comment]
@wedamija Yea, that's a good idea. I don't think they're useful at that point.

File: src/sentry/models/apitoken.py [hunk 1]
```diff
@@ -43,6 +43,7 @@ class ApiToken(Model, HasApiScopes):
     expires_at = models.DateTimeField(
         null=True, default=default_expiration
     )
+    refreshed = models.NullBooleanField()
```
Comment (markstory) [inline src/sentry/models/apitoken.py]: Do you need this if you're deleting the row?

Comment (mnoble) [reply src/sentry/models/apitoken.py]: Nope. Good call.

**Review by markstory — APPROVED** [review]

**Pull request merged**